.cirrus.yml

```diff
@@ -25,12 +25,21 @@ lint_task:
 linux_task:
   auto_cancellation: true
 
-  env:
-    matrix:
+  
+  matrix:
+    env:
       PY_VER: "3.6"
+      PYTEST_FLAGS: ""
+    env:
       PY_VER: "3.7"
+      PYTEST_FLAGS: ""
+    env:
       PY_VER: "3.8"
-
+      PYTEST_FLAGS: ""
+    env:
+      PY_VER: "3.8"
+      PYTEST_FLAGS: --async_only
```

Comment:
May be a nice way to do this? But this basically adds a new pass which is `async_conly` which means run all the tests with async on, except skip any test marked `sync_only` since we know they would fail in async mode.